New log file opened at Thu Nov 26 18:23:50 CET 2015 by 13368@Obsidian
Taking next connection to: 56926
New client attempting to connect on port: 56927-56926
  Address = /192.168.178.14:56926
This client on port 56927 has uuid f31bcafb-251f-4f42-9008-2c5ee730eab3 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-f31bcafb-251f-4f42-9008-2c5ee730eab3
Taking next connection to: 56926
New client attempting to connect on port: 56928-56926
  Address = /192.168.178.14:56926
Handshake exception on socket: 56928
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 56926
New client attempting to connect on port: 56929-56926
  Address = /192.168.178.14:56926
This client on port 56929 has uuid a1693bc3-1393-4213-912b-151b4e9c2d81 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-f31bcafb-251f-4f42-9008-2c5ee730eab3, activator-a1693bc3-1393-4213-912b-151b4e9c2d81
Taking next connection to: 56926
New client attempting to connect on port: 56939-56926
  Address = /192.168.178.14:56926
Handshake exception on socket: 56939
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 56926
New client attempting to connect on port: 56940-56926
  Address = /192.168.178.14:56926
This client on port 56940 has uuid 8a4b5de8-975b-46d7-a3b3-c7103665a064 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-a1693bc3-1393-4213-912b-151b4e9c2d81, play-fork-run-8a4b5de8-975b-46d7-a3b3-c7103665a064
Taking next connection to: 56926
